lib/win32/certstore/store_base.rb

```diff
@@ -101,9 +106,9 @@ def lookup_error(failed_operation = nil)
       # A certificate can be converted with `openssl x509 -in example.crt -out example.der -outform DER`
       def read_certificate_content(cert_path)
         unless (File.extname(cert_path) == ".der")
-          temp_file = shell_out("powershell.exe -Command $env:temp").stdout.strip.concat("\\TempCert.der")
-          shell_out("powershell.exe -Command openssl x509 -in #{cert_path} -outform DER -out #{temp_file}")
-          cert_path = temp_file
+          temp_file_path = shell_out_command("powershell.exe -Command $env:temp").stdout.strip.concat("\\TempCert.der")
```

Comment:
I brought this up on another PR but I think I merged it before we had a chance to fix i

You can get the TEMP variable from `ENV['TEMP']` without incurring all the cost of loading powershell. Only load powershell when we can't get something through FFI.

That said, it's even easier and better practice to use `temp_filepath = Tempfile.new('TempCert.der')`. This both creates a temporary file in the system temp path, but also adds some randomish characters to the name so you know you're getting a file that doesn't already exist.